pom.xml

```diff
@@ -515,6 +516,51 @@
       </build>
     </profile>
     <profile>
+      <id>spark-2.2</id>
+      <properties>
+        <spark.version>2.2.0</spark.version>
+        <scala.binary.version>2.11</scala.binary.version>
+        <scala.version>2.11.8</scala.version>
+      </properties>
+      <modules>
+        <module>integration/spark2</module>
+        <module>examples/spark2</module>
```

Comment:
@chenliang613 I have raised a pr to add package integration/spark2.2 and examples/spark2.2, we should change these two lines to spark2.2?